plugins/tickets.py

```diff
@@ -0,0 +1,1821 @@
+                'message',
+                timeout=300,
+                check=lambda msg_: (msg_.channel == msg.channel) and (msg_.author == msg.author)
+            )
+        except asyncio.TimeoutError:
+            await msg.channel.send("Note prompt timed out, please try again.")
+        if message.content.lower() == 'c':
+            await msg.channel.send("Note prompt cancelled, no note was created.")
+        else:
+            note = message.content
+    elif isinstance(maybe_note_arg, commands.StringArg):
+        note = maybe_note_arg.text
+    else:
+        # TODO: Usage
+        note = None
+
+    if note is not None:
+        # Create the note ticket
+        ticket = await create_ticket(
+            type=TicketType.NOTE,
+            modid=msg.author.id,
+            targetid=targetid,
+            created_at=datetime.datetime.utcnow(),
+            created_by=msg.author.id,
+            stage=TicketStage.COMMENTED,
+            status=TicketStatus.IN_EFFECT,
+            comment=note
+        )
+
+        # Ack note creation
+        await msg.channel.send(
+            embed=discord.Embed(description="[#{}]({}): Note created!".format(ticket.id, ticket.jump_link))
+        )
+
+
+@commands.command("tickets")
+@commands.command("ticket")
+@priv.priv("mod")
+async def cmd_ticket(msg: discord.Message, args):
+    user = msg.author
+    reply = msg.channel.send
+    no_mentions = discord.AllowedMentions.none()
+
+    cmd_arg = args.next_arg()
+    if not isinstance(cmd_arg, commands.StringArg):
+        return
+    cmd = cmd_arg.text.lower()
+
+    if cmd == "top":
+        """
+        Usage: ticket top
+        DM you the ticket at the top of your queue (if any).
+        Re-deliver the ticket at the top of your queue to your DMS.
+        """
+        mod = get_or_create_mod(user.id)
+        if not mod.current_ticket:
+            await reply("Your queue is empty, good job!")
+        else:
+            await mod.deliver()
+            if msg.channel.type != discord.ChannelType.private:
+                await reply("Ticket #{} has been delivered to your DMs.".format(mod.current_ticket.id))
+    elif cmd == "queue":
+        """
+        Usage: ticket queue [modmention]
+        Show the specified moderator's (or your own) ticket queue.
+        """
+        modarg = args.next_arg()
+        if modarg is None or isinstance(modarg, commands.UserMentionArg):
+            modid = modarg.id if modarg is not None else user.id
+            embeds = None
+            if modid in _ticketmods:
+                mod = _ticketmods[modid]
+                tickets = mod.queue
+
+                embeds = summarise_tickets(
+                    *tickets,
+                    title='Queue for {}'.format(modid),
+                    fmt="[#{id}]({jump_link}): ({status}) **{type}** for <@{targetid}>"
+                )
+
+            if embeds:
+                await pager(msg.channel, *(Page(embed=embed) for embed in embeds))
```

Comment:
I think I would prefer to see all of then at once... I was thinking we might go for a compact display of tickets to fit multiple in a message. Same for `tickets show` and `tickets showhidden`

Reply:
What if `pager` had a react that removed the pagination and sent you everything as a bunch of messages instead.

Reply:
That works :+1: